adr/0013-home-assistant-container.md

```diff
@@ -0,0 +1,71 @@
+# 0012. Installation method: Home Assistant Container
+
+Date: 2020-06-03
+
+## Status
+
+Accepted
+
+## Context
+
+Define a supported installation method as per [ADR-0012](https://github.com/home-assistant/architecture/blob/master/adr/0012-define-supported-installation-method.md).
+
+## Decision
+
+This is for running just the Home Assistant Core application on native OCI compatible containerization system. It does not provide the full Supervisor experience, and thus does not provide the Supervisor panel and add-ons.
+
+This is a general installation method that is recommended as an alternative to the Home Assistant One installation method. Due to the shared image with the Home Assistant One installation method, almost all documentation applies to the Home Assistant Container as well.
```

Comment:
> alternative to the Home Assistant One installation method
> Due to the shared image with the Home Assistant One installation method

Replace One with TBC or get that ADR published first ;)

Reply:
Published: #395.

Marked hypervisors as tbd as that was blocking it.